acceptance-tests/main.test.ts

```diff
@@ -0,0 +1,130 @@
+import {describe,it,before,after} from 'mocha';
```

Comment:
Was there any particular reason to install `mocha`? Is something missing from jest that is already installed and used?

Reply:
Jest did not work OK with zombie. We commented it in the commit.